drivers/serial/uart_stm32.h

```diff
@@ -24,6 +24,7 @@ struct uart_stm32_config {
 	/* initial parity, 0 for none, 1 for odd, 2 for even */
 	int  parity;
 	const struct pinctrl_dev_config *pcfg;
+	bool single_wire;
```

Comment:
Please move this one rank above, together with `hw_flow_control` and `parity`.
Also add a comment (you can mention half-duplex for instance)

Reply:
Done

Reply:
Done